src/Pages/woosukkim/Main/Main.js

```diff
@@ -1,11 +1,492 @@
-import React from "react";
-import "./Main.scss";
-import { withRouter } from "react-router-dom";
-
-class Main extends React.Component {
-  render() {
-    return <div></div>;
-  }
+import React, { Component } from "react";
+import "../Main/Main.scss";
+
+class Main extends Component {
+	constructor() {
+		super();
+		this.state = {
+			comment: "",
+			commentList: [],
+		};
+
+		this.submitComment = this.submitComment.bind(this);
+	}
+
+	handleComment = (event) => {
+		this.setState({
+			comment: event.target.value,
+		});
+	};
+
+	submitComment = (event) => {
+		event.preventDefault();
+		this.state.commentList.push({ text: this.state.comment });
+		this.setState({ comment: "" });
+		event.target.reset();
+	};
+
+	render() {
+		const currentValue = this.state.comment;
```

Comment:
새로 변수를 만드는게 아니라 비구조화할당 적용해서 작성해주세요!

```suggestion
		const { comment } = this.state;
```